code/src/types/cart.types.ts

```diff
@@ -27,3 +27,10 @@ export type CartType = {
   items: CartItem[];
   totalPrice: number;
 };
+
+export type CartItemProps = {
+  item: CartItem;
+  onRemove: (product: CartItem) => void;
+};
```

Comment:
move this type to file with this Component. We are not defining the types for the components props inside global types